common/output.c

```diff
@@ -300,24 +300,15 @@ void logg_close(void)
 
 /*
  * legend:
- *  ! - ERROR:
- *  ^ - WARNING:
- *  ~ - normal
- *  # - normal, not foreground (logfile and syslog only)
- *  * - verbose
- *  $ - debug
- *  none - normal
- *
- *	Default  Foreground LogVerbose Debug  Syslog
- *  !	  yes	   mprintf     yes      yes   LOG_ERR
- *  ^	  yes	   mprintf     yes	yes   LOG_WARNING
- *  ~	  yes	   mprintf     yes	yes   LOG_INFO
- *  #	  yes	     no	       yes	yes   LOG_INFO
- *  *	  no	   mprintf     yes	yes   LOG_DEBUG
- *  $	  no	   mprintf     no	yes   LOG_DEBUG
- *  none  yes	   mprintf     yes	yes   LOG_INFO
+ * NAME     OLD        EXPLAIN
+ * INFO     none or ~   normal
+ * INFO_NF  #           normal, no foreground
+ * DEBUG    *           debug, verbose
+ * DEBUG_NV $           debug, non-verbose
+ * WARNING  ^           warning
+ * ERROR    !           ERROR
  */
```

Comment:
The "OLD" column probably isn't needed but it sure helps for this review. 

The `_NF` and `_NV` suffixes aren't intuitive to me but the "EXPLAIN" column helps.  I think "`not foreground (logfile and syslog only)`" would be a little better than just "`no foreground`" to explain the `_NF` variant. 

Please also update the copy in `output.h`. You may even consider moving this comment block to `output.h` just above the `logg()` function prototype so it's more obvious in the future that the comment exists for both the function and the enum.  I do like having a copy for the function for easy reference when i highlight `logg` in my editor.  Maybe the function-version should also explain the return value.

Reply:
I updated and moved the comment block to `output.h`, please review the comments again to determine if I got you right